IntelliJKite/src/KiteProjectComponent.java

```diff
@@ -104,29 +104,37 @@ public void initComponent() {
         ApplicationManager.getApplication().invokeLater(new Runnable() {
             @Override
             public void run() {
+            if(m_project != null) {  // double-fix for a user-report NullReferenceException here
```

Comment:
do you need to check it `m_kiteConnection` is `null` as well? looks like it's being used in `sendEvent`